tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Support/ContactReferenceList.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Entities;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Support
+{
+    internal sealed class ContactReferenceList
```

Comment:
I feel like we could make a cool generic class for these variants, save that for tech debt though

Reply:
Yeah good shout 👍